bundler/lib/bundler/cli.rb

```diff
@@ -316,28 +316,11 @@ def update(*gems)
       D
       method_option "paths", :type => :boolean,
                              :banner => "List the paths of all gems that are required by your Gemfile."
+      method_option "verbose", :type => :boolean,
+                               :banner => "Show verbose output."
```

Comment:
The verbose flag is global and not specific to any command, so there's no need to specify it here. It was a mistake that the deprecation message also applied to the `--verbose` flag.

As an extra improvement, could you change [the docs](https://github.com/rubygems/rubygems/blob/8e204557a657d3a65f40f4b1598d92ab2adcbca8/bundler/lib/bundler/man/bundle-show.1.ronn) to document the missing `--outdated` flag? (You'll need to run `bin/rake man:build` after that to regenerate the other man pages after that). Alternative, since `bundle show --outdated` is not really useful IMO, we could keep the deprecation for that. What do you think?

Reply:
Thanks for the information about `--verbose`. Will add the deprecation for `--outdated` since there's `bundle outdated` already as you told me.

Reply:
I agree restoring the deprecation of `bundle show --outdated` is best :+1:.